docs/local-testing.md

```diff
@@ -4,258 +4,212 @@ In this walkthrough, you will learn how to use the kCTF infrastructure.
 
 In detail, this means:
 
-1. Setting the right `umask`.
-2. Deciding on your test environment ([Docker](#testing-with-docker-only) or [Kubernetes](#testing-with-kubernetes)) and performing the related steps.
-3. [Debugging](#debug-failures) any errors you encountered. 
+1. [First-time setup](#first-time-setup): Setting the right `umask`, installing dependencies, enabling user namespaces.
+2. [Using kCTF](#using-kctf): Downloading the SDK, creating a local cluster and creating tasks.
+3. [Troubleshooting](#Troubleshooting): Basic `kctf chal debug` commands, and `kubectl` usage.
 
-## Set the right umask
+## First-time setup
+
+Following this walkthrough requires a local Linux machine capable of running Docker.
+
+### Set the right umask
 
 Since nsjail will run commands as another user, the challenge files need to be readable by all users. To enable this, set the following umask:
-```
+```bash
 umask a+rx
 ```
 
-## Testing with Docker only
+This should also be setup in our `.bashrc` and `.bash_profile` to ensure you never forget to set it.
```

Comment:
not sure if we should advise people to reduce their security. If you have a restrictive umask like that there's probably a reason

Reply:
it's necessary though. otherwise kctf should be compatible with restrictive umasks and do chmod on the Dockerfile for files that end up behind nsjail

Reply:
but we already warn on kctf/activate. So if you forget about it, we will tell you

Reply:
I'll remove it, but `kctf activate` is a bit late. this will cause issues with gctf I think =/